docs/best-practices/folders-structure.md

```diff
@@ -80,3 +90,39 @@ The main difference between a KMM and a native project is that the presentation
             │   └── model
             └── FeatureProvider.kt # DI Component. Provides interactor mostly
 ```
+</TabItem>
+
+<TabItem value="angular">
+
+### Angular Project structure
+
+```txt
+.
+└── project
+    └── src
+        ├── app                      # Angular stuff
+        │   ├── feature-name         # UI module
+        │   │   ├── component-name   # Includes component + html + css + view-state + test
```

Comment:
**Nit:** Do we want to make any distinction between route, smart, dumb… components?

Reply:
At this moment reusable components are placed in shared/components. But components that are strictly inside a feature, should be placed inside the ui-feature folder no matter if they are smart or dumb... don't you think? Do you have a case where we need to distinguish a component in any other way?